create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/utils.py

```diff
@@ -50,6 +50,10 @@ def get_config(argv: Any = None) -> Any:
     return commandline.config_from_options(options, CONFIG_TRAFARET)
 
 
-def init_config(app: web.Application, *, config: list = None) -> None:
+def init_config(
+        app: web.Application,
+        *,
+        config: Optional[Dict[str, Dict[str, str]]] = None
```

Comment:
Why dict, not list?

Reply:
Because config comes comes in a structure like this:

`{'postgres': {'host': 'host', 'user': 'user'}}`

Reply:
Look at the get_config signature. It accepts a list of `argv` and then feeds that into `argparse`